prisma/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://tirusTimes:1234@localhost:5432/prisma-postgres?schema=public
```

Comment:
Please add newline at EOF.

Suggestion: add to the project the `.editorconfig` with the content:

```
# editorconfig.org
root = true

[*]
end_of_line = lf
insert_final_newline = true
```